TypeCobol/Compiler/File/SourceFileProvider.cs

```diff
@@ -30,6 +30,14 @@ public SourceFileProvider()
             CobolLibraries = new List<ICobolLibrary>();
         }
         
+        /// <summary>
+        /// Remove all Cobol Libraries
+        /// </summary>
+        public void RemoveAllLibraries()
+        {
+            CobolLibraries.Clear();
+        }
```

Comment:
Not needed

Reply:
Removed in [8d9c2da](https://github.com/TypeCobolTeam/TypeCobol/pull/2158/commits/8d9c2daba074a18dbf5260d351d3278d470ebf44)